packages/service/src/domain/commands/processors/skip-command.ts

```diff
@@ -2,17 +2,32 @@ import CommandDefinition from '../model/command-definition';
 import CommandProcessingResponse, { makeSingleTextProcessingResponse } from '../model/command-processing-response';
 import * as EventStreamService from '../../../event-stream/event-stream-service';
 import { SkipEvent } from '../../../event-stream/model/events';
+import Command from '../model/command';
 
-async function skipCommandProcessor() : Promise<CommandProcessingResponse> {
-  const event: SkipEvent = { id: 'SkipEvent' };
+async function skipCommandProcessor(command: Command) : Promise<CommandProcessingResponse> {
+  const all = command.rawArgs === 'all';
+  let amount = parseInt(command.rawArgs, 10);
+
+  if ((Number.isNaN(amount) || amount < 1) && !all) {
+    throw new Error('Skip akceptuje liczby naturalne lub argument "all"');
+  }
+
+  amount = amount || 1;
+
+  const event: SkipEvent = { id: 'SkipEvent', all, amount };
```

Comment:
możesz tutaj zrobić
```suggestion
  const event: SkipEvent = { id: 'SkipEvent', all, amount: amount || 1 };
```
i wtedy nie musisz robić `let` i nowego przypisania